pep-0705.rst

```diff
@@ -2,7 +2,7 @@ PEP: 705
 Title: TypedMapping: Type Hints for Mappings with a Fixed Set of Keys
 Author: Alice Purcell <alicederyn@gmail.com>
 Sponsor: Pablo Galindo <pablogsal@gmail.com>
-Discussions-To: https://mail.python.org/archives/list/python-dev@python.org/thread/2P26R4VH2ZCNNNOQCBZWEM4RNF35OXOW/
+Discussions-To: pending
```

Comment:
Just FYI, the linter will go red on this and flag it, but perhaps that's intentional as a reminder to update it immediately prior to merging (if that's your plan).